requirements_test.txt

```diff
@@ -12,6 +12,7 @@ mypy==0.812
 pre-commit==2.12.1
 pylint==2.8.0
 astroid==2.5.5
+mccabe==0.6.1
```

Comment:
This file includes the pre-commit version (preventing to maintain additional locations), thus can be removed.

```suggestion
```